this patch is needed in order to run servers on fly.io. due to their anycast setup, they tunnel and encrypt the UDP traffic.
see https://fly.io/docs/app-guides/udp-and-tcp/#you-might-need-to-be-mindful-of-mtus

diff --git src/enet/include/enet/enet.h src/enet/include/enet/enet.h
index fc45cbd..2246327 100644
--- src/enet/include/enet/enet.h
+++ src/enet/include/enet/enet.h
@@ -212,7 +212,7 @@ enum
    ENET_HOST_RECEIVE_BUFFER_SIZE          = 256 * 1024,
    ENET_HOST_SEND_BUFFER_SIZE             = 256 * 1024,
    ENET_HOST_BANDWIDTH_THROTTLE_INTERVAL  = 1000,
-   ENET_HOST_DEFAULT_MTU                  = 1400,
+   ENET_HOST_DEFAULT_MTU                  = 1300,
    ENET_HOST_DEFAULT_MAXIMUM_PACKET_SIZE  = 32 * 1024 * 1024,
    ENET_HOST_DEFAULT_MAXIMUM_WAITING_DATA = 32 * 1024 * 1024,

